Formula/hashcat.rb

```diff
@@ -36,3 +56,17 @@ def install
     system testpath/"hashcat --benchmark -m 0 -D 1,2 -w 2"
   end
 end
+
+__END__
+diff --git a/src/filehandling.c b/src/filehandling.c
+index 345d451..05baae9 100644
+--- a/src/filehandling.c
++++ b/src/filehandling.c
+@@ -5,6 +5,7 @@
+ 
+ #include "common.h"
+ #include "types.h"
++#include "limits.h"
```

Comment:
Can this be reported to upstream?

Reply:
Yes, I wanted to make sure it passed CI on all platforms and now that it has I am going to submit it and link the PR in a comment.